### PR TITLE
Address Alexey DISCUSS items

### PR DESCRIPTION
**Pull request by richsalz**
Also ask IANA to create the trans:error registry.

Also tweak some markdown syntax so my vim coloring isn't a screen
full of yellow :)

### CODE REVIEW

File: draft-ietf-trans-rfc6962-bis.md [hunk 1]
```diff
@@ -2105,6 +2105,50 @@ details in this registry.
...
+Specification: RFCXXXX
+
+Repository: https://www.iana.org/assignments/trans
+
+Index value: No transformation needed.
+
+### TRANS Error Types
+
+IANA is requested to create a new registry for errors.
+Requirements for this registry are specification required.
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md:2125]: Nit: s/specification required/Specification Required/

File: draft-ietf-trans-rfc6962-bis.md [hunk 2]
```diff
@@ -2105,6 +2105,50 @@ details in this registry.
...
+| reference  | string | RFCXXXX   |
+|------------+--------+-----------|
+
+The initial values are as follows, taken from the text above:
+
+|------------+----------------------------------+-----------|
+| Identifier | Meaning                          | Reference |
+|------------+----------------------------------+-----------|
+| malformed  | The request could not be parsed. | RFCXXXX   |
+|------------+----------------------------------+-----------|
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md:2143]: What's this line for?

File: draft-ietf-trans-rfc6962-bis.md [hunk 3]
```diff
@@ -2105,6 +2105,50 @@ details in this registry.
...
+| badSubmission | `submission` is neither a valid certificate nor a valid precertificate        | RFCXXXX |
+| badType       | `type` is neither 1 nor 2     | RFCXXXX |
+| badChain      | The first element of `chain` is not the certifier of the `submission`, or the second element does not certify the first, etc. | RFCXXXX |
+| badCertificate | One or more certificates in the `chain` are not valid (e.g., not properly encoded) | RFCXXXX |
+| unknownAnchor  | The last element of `chain` (or, if `chain` is an empty array, the `submission`) both is not, and is not certified by, an accepted trust anchor | RFCXXXX |
+| shutdown       | The log is no longer accepting submissions | RFCXXXX |
+|---------------+----------------------------------|
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md:2150]: This covers the error codes from sections 5 and 5.1, but misses several additional error codes defined in other 5.x sub-sections.

**Comment by richsalz** [comment]
pushed a commit that fixes the typo's and adds the missing errros as noted by @robstradling

**Pull request merged**